src/SDKs/_metadata/sql_resource-manager.txt

```diff
@@ -1,11 +1,14 @@
-2018-04-18 23:35:28 UTC
+
+Type dsc to start/install the DS Consolidated Console
+
+2018-05-21 20:53:08 UTC
 
 1) azure-rest-api-specs repository information
-GitHub user: jugeorge
+GitHub user: Azure
 Branch:      master
-Commit:      d08c7f54a125819caaa8c5f553206d1adbae60f9
+Commit:      c6fea13a449c7e60d515759d687f18c9079c8522
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\jugeorge\AppData\Roaming\npm `-- autorest@2.0.4262  
-Latest installed version:    
```

Comment:
Please generate the code using [this](https://github.com/jaredmoo/azure-sdk-for-net/blob/8dd820d23bf7b9f5bd35e2a576681b7559af35f8/src/SDKs/SqlManagement/Management.Sql/generate.ps1) script instead